src/routes/(site)/hero.svelte

```diff
@@ -14,7 +14,70 @@
       </div>
     </div>
 
-    <img src="/assets/capy-power.svg" alt="Chip the Capybara surrounded by all ACM Team badges" />
+    <map name="capy-power">
+      <area
+        alt="General Team"
+        title="General Team"
+        href="/teams#general"
+        coords="200,10 160,50 200,90 240,50 "
+        shape="polygon"
+      />
+      <area
+        alt="AI Team"
+        title="AI Team"
+        href="/teams#ai"
+        coords="330,60 290,100 330,140 370,100"
+        shape="polygon"
+      />
+      <area
+        alt="Design Team"
+        title="Design Team"
+        href="/teams#design"
+        coords="370,170 330,210 370,250 410,210"
+        shape="polygon"
+      />
+      <area
+        alt="Dev Team"
+        title="Dev Team"
+        href="/teams#dev"
+        coords="330,290 290,330 330,370 370,330"
+        shape="polygon"
+      />
+      <area
+        alt="Algo Team"
+        title="Algo Team"
+        href="/teams#algo"
+        coords="200,340 160,380 200,420 240,380"
+        shape="polygon"
+      />
+      <area
+        alt="ICPC Team"
+        title="ICPC Team"
+        href="/teams#icpc"
+        coords="80,290 40,330 80,370 120,330"
+        shape="polygon"
+      />
+      <area
+        alt="Open Source Software Team"
+        title="Open Source Software Team"
+        href="/teams#oss"
+        coords="50,170 10,210 50,250 90,210"
+        shape="polygon"
+      />
+      <area
+        alt="Game Dev Team"
+        title="Game Dev Team"
+        href="/teams#gamedev"
+        coords="90,60 50,100 90,140 130,100"
+        shape="polygon"
+      />
+    </map>
+
+    <img
```

Comment:
Caught by @karnikaavelumani:

![image](https://github.com/EthanThatOneKid/acmcsuf.com/assets/31261035/2ca260c2-a943-4f3a-be4d-165a2ecae463)

Reply:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/1011/commits/3fb1fa5082f802bced2515922380363836d466f1>.